components/wb-bubble/wb-bubble-va-en.html

```diff
@@ -0,0 +1,59 @@
+---
+title: Bubble - Open Overlay
+language: en
+category: Plugins
+description: Working example of the virtual assistant bubble with customized avatar.
+altLangPage: wb-bubble-fr
+dateModified: 2021-06-21
+---
+<span class="wb-prettify all-pre hide"></span>
+
```

Comment:
Put the following sentence in a `p` right after the H1 here, since it is very important: 

> The overlay markup must be next to the button in order to ensure a proper focus order when the overlay are closed by the user.